### DISCUSSION TITLE
please help!! update glitch :(

### DISCUSSION THREAD

**Discussion by strawberrybooba** [Monika-After-Story/MonikaModDev#8409] [QA - tech support + general]
https://user-images.githubusercontent.com/91277366/145118851-9f044c13-f4f5-4097-a1a0-c28397c0ec61.MOV

i got past this before but i had an unfortunate event happen and i had to restore a backup from... before i got past the update glitch,, ill make sure to save it next time! i took that video 10 minutes ago and its still broken :( when i hit cancel, it crashes the game too

**Comment by ThePotatoGuy** [comment]
Try to update again later.
